engine/lib/friends.php

```diff
@@ -87,6 +87,7 @@ function _elgg_friends_setup_user_hover_menu($hook, $type, $return, $params) {
 			// Always emit both to make it super easy to toggle with ajax
 			$return[] = \ElggMenuItem::factory(array(
 				'name' => 'remove_friend',
+				'data-toggle' => 'add_friend',
```

Comment:
i went for `toggle` in the factory.... in elgg_view_menu_item this gets translated to `data-toggleable-item`